pkg/auth/kfm_claims.go

```diff
@@ -26,6 +27,13 @@ func (c *KFMClaims) GetAccountId() (string, error) {
 	return "", fmt.Errorf("can't find '%s' attribute in claims", tenantUserIdClaim)
 }
 
+func (c *KFMClaims) GetClientId() (string, error) {
```

Comment:
We could use the method here as well: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/22024f07663adf924c1054b3d255899d6b3b2e3b/pkg/auth/operator_authz_middleware.go#L37